ycms/cms/templates/_base.html

```diff
@@ -53,89 +53,93 @@
             {% endfor %}
         </div>
     {% endif %}
-    <div id="sidebar" class="absolute h-full w-64 bg-white border-r shadow-md">
-        <a href="{% url 'cms:protected:index' %}" class="block border-b">
-            <img src="{% static 'images/logo-ycms.png' %}"
-                 width="200px"
-                 height="auto"
-                 alt="{% translate " Logo" %}"
-                 class="my-3 mx-auto">
-        </a>
-        <ul>
-            <span class="block mt-5 px-5 text-gray-500 tracking-wide">{% translate "General" %}</span>
-            {% if perms.cms.add_ward %}
+    <div class="lg:flex">
+        <div id="sidebar"
+             class="fixed h-full w-64 bg-white border-r shadow-md hidden lg:block">
```

Comment:
```suggestion
             class="fixed h-full w-64 bg-white border-r shadow-md transform -translate-x-full lg:translate-x-0">
```
This way we can use the one sidebar on desktop and mobile, plus `translate-x` is animateable! :D